### ISSUE TITLE
vjs-fluid with square poster art for audio defaults to 16:9 aspect ratio

### ISSUE THREAD

**Issue by classicjazz** [videojs/video.js#4358]
I am displaying square poster art for internet radio stations with videojs using the audio tag.  I am using videojs 6.0.1.

See http://radio.mikeconnelly.net

If I omit the vjs-fluid style, it correctly renders the square poster art but it is not responsive. 

If I include the vjs-fluid style, it is responsive but defaults to a 16:9 aspect ratio, padding the left and right sides of the poster with black bars. 

Is there a function call that I am missing or is this a limitation of 6.0.1?

**Comment by gkatsev** [comment]
Video.js defaults to 16:9 if it doesn't know what the aspect ratio is by reading the video. An audio file doesn't have associated video width and height. You should be able to tell videojs the aspect ratio via the [`aspectRatio()`](http://docs.videojs.com/Player.html#aspectRatio) method (`player.aspectRatio('4:3')`) or by adding the `.vjs-4-3` class to the player.

Hope that helps.

**Comment by classicjazz** [comment]
So, I think I solved this for responsive square poster art for audio. 

For other readers, here's what I did:

Only use the following class (don't reference fluid, .vjs-4-3, etc):  
class="video-js vjs-big-play-centered" 

In data-setup, specify the following:
data-setup='{
    	"aspectRatio": "1:1"
    			}'>

(don't mention fluid=true). Specifying the aspect ratio will make it responsive. If you don't specify the aspect ratio, it will be square by default but not responsive.
